pages/common/gox.md

```diff
@@ -0,0 +1,19 @@
+# gox
+
+> A tool for cross-compiling Go programs.
```

Comment:
Please add the home page url in a new line after this.

Reply:
Like this:

```markdown
> Homepage: <https://example.com>.
```